addons/binding/org.openhab.binding.gpio/src/main/java/org/openhab/binding/gpio/handler/PigpioRemoteBridgeHandler.java

```diff
@@ -0,0 +1,83 @@
+/**
+ *
+ */
+package org.openhab.binding.gpio.handler;
+
+import static org.openhab.binding.gpio.GPIOBindingConstants.*;
+
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import jpigpio.JPigpio;
+import jpigpio.PigpioException;
+import jpigpio.PigpioSocket;
+
+/**
+ * @author nils
```

Comment:
Please add / check whether each class has a meaningful description of its purpose.